MetaMask/metamask-extension
# feat: Add MetaMask Portfolio What's New notification

## **Description**
Adding a what's new notification for Portfolio



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23359?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2181

## **Manual testing steps**

1. Open Extension
2. Confirm that the Portfolio What's New appears
3. Confirm that upon clicking the CTA, you are sent to the Portfolio website
4. Confirm that on dismissal, the notification does not show up again

## **Screenshots/Recordings**



### **Before**



### **After**


<img width="358" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/44d9d700-1a46-4595-a126-0f95ce41e3fe">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
